src/services/http-client.js

```diff
@@ -3,7 +3,7 @@ import store from '@/store'
 import tokenService from '@/services/token'
 
 const options = {
-  baseURL: process.env.VUE_APP_API_HOST,
+  baseURL: "https://pdrc-calculator-strapi.herokuapp.com",
```

Comment:
no no no)

Reply:
we used this link because it dont`t work else. but it works now. wonders!